packages/eslint-tool/lib/index.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
+
+const config = {
+	env: {
+		browser: true,
+		es6: true,
```

Comment:
Was reading through the eslint docs and didn't know if this should be bumped up to the latest [es2022](https://eslint.org/docs/latest/use/configure/language-options) or leave as es6?

Reply:
I'm unsure, maybe we don't get to make this decision because each project should get to decide? But really don't know. Maybe requires more research to see how it works when you apply multiple configs on top of each other

Reply:
Ok will look further into this :)

Reply:
Decided to bump up to es2022 as (from what I've read) this means that this environment will provide predefined global variables from ECMAScript 2022 globals.

Which I interpret as 🤔 : any new global variables that have been released since es6 (ie pre-2016) up to es2022 would also be included in the ESLint check. But correct me if I've interpreted this incorrectly 🙇🏽‍♀️